src/stories/ui/button-group/ButtonGroup.stories.tsx

```diff
@@ -0,0 +1,78 @@
+import type { Meta, StoryObj } from '@storybook/react'
+import { ButtonGroup } from '../../../sections/ui/button-group/ButtonGroup'
+import { Button } from '../../../sections/ui/button/Button'
+import { ButtonToolbar } from '../../../sections/ui/button-group/ButtonToolbar'
+import { DropdownButton } from '../../../sections/ui/dropdown-button/DropdownButton'
+import { DropdownButtonItem } from '../../../sections/ui/dropdown-button/dropdown-button-item/DropdownButtonItem'
+
+/**
+ * ## Description
+ * A ButtonGroup is a wrapper component for grouping multiple buttons together, providing them with a common layout and
+ * spacing. It can be used to visually indicate a related set of actions, or to improve the layout of multiple buttons
+ * on a page.
+ *
+ * ## Usage guidelines
+ *
+ * ### Dos
+ * - Use ButtonGroup to group related actions together.
```

Comment:
It's a little difficult to see where one button stops and another begins:

![Screen Shot 2023-04-25 at 5 16 40 PM](https://user-images.githubusercontent.com/21006/234406062-6003a453-9e18-4b43-bafd-64a3f2159ea9.png)

Would it be possible to have some space or a line between them, like the old design? Here's a screenshot:

![Screen Shot 2023-04-25 at 5 16 53 PM](https://user-images.githubusercontent.com/21006/234406061-618a2ca6-3fd0-4bee-8b0f-de5f7294481d.png)

Reply:
Ok, I added the border! I couldn't find a way to customize the border using Bootstrap sass variables then I added a CSS class

![image](https://user-images.githubusercontent.com/23359572/234894211-6f2a6a0b-95c2-48bc-9a28-06fba4cfe6e3.png)